tests/hash-backend-integration/src/tests/subgraph/friendship.test.ts

```diff
@@ -183,6 +188,68 @@ afterAll(async () => {
   await resetGraph();
 });
 
+describe("Ontology queries", () => {
+  it("read data types", async () => {
+    const subgraph = await getDataTypes(graphContext, {
+      query: {
+        filter: {
+          all: [],
+        },
+        graphResolveDepths: {
+          ...zeroedGraphResolveDepths,
+          inheritsFrom: { outgoing: 1 },
+        },
+        temporalAxes: fullDecisionTimeAxis,
+      },
+    });
+    expect(subgraph.roots.length).toEqual(3);
+    expect(Object.keys(subgraph.edges).length).toEqual(0);
+    const data_types = getRoots(subgraph);
+
+    expect(data_types.length).toBe(3);
```

Comment:
Do we want to check equality (or at least the `$id`) of the types in here to be what we expect? 

Not suggesting it is worth the extra code, but having an error about a specific vertex being missing (or making sure we don't accidentally put the wrong vertices in the wrong places, etc.) might be more helpful than just checking a length. Happy to merge otherwise

Reply:
A few extra lines don't hurt here: [`1d7c84c` (#2641)](https://github.com/hashintel/hash/pull/2641/commits/1d7c84c1ed389871bec4f5e81d0bcfaaf4ce1164)

Reply:
Thanks, I prefer this :)

Although we probably want to improve this a bit later to avoid the magic strings and have them as a constant that's referable somewhere, or something we can import from the underlying test data. Could see this being a bit annoying to update if we change the tests

Reply:
I don't know yet how we could achieve this as we cannot easily use the strings from the snapshot, but I agree that these should be improved at some point.